### DISCUSSION TITLE
No Ding Events

### DISCUSSION THREAD

**Discussion by mastermc0** [tsightler/ring-mqtt#620] [Support Q&A] [answered]
I'm not seeing the "Ding" state topic changing to on the motion state.  When I first configured the container it seemed fine but for the last 24 hours or so it's not worked.   I can see new "OFF" entries in the log however around motion/ding events.

```
ring-mqtt  | 2023-04-28T16:38:32.384Z ring-mqtt [Back Door] ring/******/camera/******/motion/state OFF
ring-mqtt  | 2023-04-28T16:38:32.384Z ring-attr [Back Door] ring/******/camera/******/motion/attributes {"lastMotion":1682699639,"lastMotionTime":"2023-04-28T16:33:59Z","personDetected":false,"motionDetectionEnabled":true}
ring-mqtt  | 2023-04-28T16:38:32.384Z ring-mqtt [Back Door] ring/******/camera/******/ding/state OFF
ring-mqtt  | 2023-04-28T16:38:32.384Z ring-attr [Back Door] ring/******/camera/******/ding/attributes {"lastDing":1682699418,"lastDingTime":"2023-04-28T16:30:18Z"}
```

**Comment by mastermc0** [comment]
Don't know if this is related seeing it in both ring-mqtt container and when I tried the Homebridge plugin

```
(node:6926) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 ON_READY listeners added to [PushReceiver]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
```

**Reply by tsightler** [reply, marked as answer]
Assuming you have more than 10 cameras then I would consider this message "normal" as NodeJS just assumes that more than 10 subscriptions to an event is somehow "wrong" and issues this warning but it doesn't actually stop anything from working.  The setmaxlisters just increases the level where the warning is emitted.

If you are seeing the same behavior in both ring-mqtt and ring-homebridge then you need to figure out why you are not receiving notifications.  I can't really help with this but the ring-mqtt FAQ already documents the most common reasons (Smart Alerts) but there are also cases where, for unknown reasons, Ring just stops sending push notifications to a specific app.  The only fix for this latter item is to remove the account from authorized devices and re-authenticate.  See https://github.com/dgreif/ring/issues/933#issuecomment-1373685385.

**Comment by pepepere** [comment]
No ding event in all my doorbells, motion event and others are working.
API has changed?

**Reply by Milinator** [reply]
Same for me, have a look here: https://github.com/dgreif/ring/issues/1212#issue-1698577948

**Comment by mastermc0** [comment]
I found my issue was due to some domain filtering I have on my network that was blocking mtalk.google.com which is used for notifications. 

Both ding and motion are working fine for me now so if you’re having another issue I’d suggest opening a new discussion as my original issue was not due to this repo rather my own local network confitg.
